bindings/go/runtime/go.mod

```diff
@@ -4,16 +4,21 @@ go 1.24.1
 
 require (
 	github.com/cyberphone/json-canonicalization v0.0.0-20241213102144-19d51d7fe467
+	github.com/invopop/jsonschema v0.13.0
```

Comment:
Worried about this dependency, are we sure on its stability?

Reply:
I'm not. Seems like a stable library, being used by quite a lot of people. There is precious else we can do other than try to implement everything ourselves. That would be rather daunting.